### ISSUE TITLE
Add a Demo for onDelete option

### ISSUE THREAD

**Issue by etherealm13** [ajainarayanan/react-dag-app#1]
Unable to figure out how to make onDelete work based on the example. Could you add the onDelete option on any nodes ? @ajainarayanan

**Comment by ajainarayanan** [comment]
@etherealm13 I guess this repo hasn't been updated with latest changes. Can you refer to this example in [react-dag repo](https://github.com/ajainarayanan/react-dag/blob/feature/2.0.0.alpha/dev/components/NodeType2/index.tsx#L51)

**Comment by etherealm13** [comment]
This is great. Thanks.
Is there a JavaScript example by any chance ? I'm not using the TS version.
or Could you just guide me as to how can I make the onDelete option work
with the JS code ? As in what portion of files need to changed ?

I would like to reiterate that this project has been a life saver for me.
Thanks a lot for all the hard work you have put into this.

**Comment by ajainarayanan** [comment]
Essentially you would do something very similar to typescript. It just wouldn't have any static types. 
For instance in the node that you want to delete you could add a "delete" or "x" button with onClick handler something like this,

```
delete = () => {
    this.props.onDelete(this.props.id, [
      `${this.props.id}-DottedEndpoint-right`,
      `${this.props.id}-DottedEndPoint`,
    ]);
  };```
Assuming your individual node is a class implementation

**Comment by etherealm13** [comment]
Thanks for the help.
Could you recommend some doc source to get the ins and outs of all the
settings and workings of react dag ? Do you think understanding of JsPlumb
will help me understand react-dag in a better way?

**Comment by ajainarayanan** [comment]
@etherealm13 Yes. React-dag is pretty much a wrapper around jsplumb. Ideally you don't need to know too much about it but if you want to configure specific things in the graph then an understanding of jsplumb definitely helps.
